preppal-fe/src/pages/profile.tsx

```diff
@@ -0,0 +1,89 @@
+import Card from 'react-bootstrap/Card';
+import Image from 'react-bootstrap/Image';
+import { Button, Col, Row, Stack } from 'react-bootstrap';
+import React from 'react';
+
+const logo = require('../assets/logo.png')
+
+function Profile() {
+    const [userName, setName] = React.useState("");
+    const [userBio, setBio] = React.useState("");
+    const [userFollowing, setFollowing] = React.useState<any[]>([]);
+    const [userFollowingCount, setFollowingCount] = React.useState(0);
+    React.useEffect(() => {
+        fillUserContent();
+    }, []);
+
+    const fillUserContent = async () => {
+        const token = localStorage.getItem("token");
+        try {
+            if (token) {
+                const req = {
+                    method: "GET",
+                    headers: {
+                        "x-auth-token": token
+                    }
+                };
+                const res = await fetch("http://localhost:9001/api/auth/", req).then(res => res.json());
+                setName(res.username);
+                setBio(res.bio);
+                setFollowing(res.following);
+                setFollowingCount(userFollowing.length)
+            }
+        } catch (err) {
+            console.error(err);
+        }
+    };
+
+    function edit(): React.FormEventHandler<HTMLLabelElement> | undefined {
+        console.log("Edit was clicked")
+        //will either go to new page for edits, or will make this page editable
+        return
+    }
```

Comment:
linter will spit a warning here when we try to do a build since edit() was not used.
` Line 38:14:  'edit' is defined but never used                                                                                          @typescript-eslint/no-unused-vars`

I think you should make the button just call the function just to fix the warning. even though you are probably going to work on this next.

Reply:
'onSubmit' has been added to call the empty function. Functionality to be added in a new branch. Ran 'npm run build' and 'npm start' to confirm.